src/kbmod_wf/tno_workflow.py

```diff
@@ -0,0 +1,277 @@
+"""This workflow definition represents the task flow that was used to prepare results
+for the TNO 2024 presentation. It has been slightly updated to also use sharded
+workunits.
+"""
```

Comment:
I think it's interesting to observe what changed between the two workflows. Except tranlsating the lst file to ic and globbing all ".collections" instead of "lst" it doesn't seem like much. 

I'd be curious to hear your take on what you think the inputs for a workflow should be and then build the tooling for that.

Reply:
You're correct, that there really isn't much that has changed between the two workflows. My take at the beginning of working on this parsl project was that the order would be roughly:

1. Reproduce the workflow that was done manually in prep for TNO
2. Polish that workflow so that more than one person could run at a time
3. Tack on a region search task to the beginning of the workflow. i.e. provide a list of search queries as input
5. Tack on filtering to the end of the workflow so that search results will go through the CNN
6. Tack on visualization prep to make examining results easier.
7. Polish for production
8. Tack on a step that accepts a catalog of newly acquired images - for each new image, determine by some criteria if there are enough images now to do a search in a particular patch of sky, and create a region search query for that patch.

So to make a long story short, in the near term ImageCollections should be the input. Then queries for a region search step. Then a catalog of images acquired since the last time the workflow was run. 

That is in order of near to long term _and_ high to low confidence. :)